examples/inspection/plot_linear_model_coefficient_interpretation.py

```diff
@@ -92,21 +93,23 @@
 _ = sns.pairplot(train_dataset, kind='reg', diag_kind='kde')
 
 ##############################################################################
-# Looking closely at the WAGE distribution it could be noticed that it has a
-# long tail and we could take its logarithm
-# to simplify our problem and approximate a normal distribution.
+# Looking closely at the WAGE distribution reveals that it has a
+# long tail. For this reason, we should take its logarithm
+# to turn it approximately into a normal distribution.
```

Comment:
Do you think it could nice to add why a normal distribution is better (I assume this because this is an assumption of the linear model, normality of the residuals?)